e2e-polybft/e2e/helpers_test.go

```diff
@@ -98,3 +99,24 @@ func getRootchainValidators(relayer txrelayer.TxRelayer, checkpointManagerAddr e
 
 	return validators, nil
 }
+
```

Comment:
Worth considering moving everything which are not e2e tests themselves from `bridge_test` here in helpers_test.go (namely: `sendExitTransaction`, `getExitProof` and `checkStateSyncResultLogs`)